### PR TITLE
Hide progress on windows

### PR DESCRIPTION
**Pull request by sanderpick**
- Don't use progress bar on windows
- Fix `buck init` (`unfreeze` flag used but not defined)

### CODE REVIEW

**Comment by sanderpick** [comment]
I'm curious why the local buckets tests never run from cache. All the other one do. It's always run on the same machine.

**Review by jsign — APPROVED** [review]

File: cmd/buck/cli/init.go [hunk 1]
```diff
@@ -73,8 +73,10 @@ Use the '--hard' flag to discard all local changes.
 			cmd.Fatal(errors.New("--cid cannot be used with an existing bucket"))
 		}
 
-		unfreeze, err := c.Flags().GetBool("unfreeze")
-		cmd.ErrCheck(err)
+		// (jsign): re-enable when this feature is usable in mainnet.
+		//unfreeze, err := c.Flags().GetBool("unfreeze")
+		//cmd.ErrCheck(err)
+		var unfreeze bool
```
Comment (jsign) [inline cmd/buck/cli/init.go:79]: I guess we need to include some test to compile all the project to catch these things.

**Comment by jsign** [comment]
> I'm curious why the local buckets tests never run from cache. All the other one do. It's always run on the same machine.

Yeah, weird. I tested locally and should be cached. Nothing popups in my mind... maybe some ghrunner touching some files.

File: cmd/buck/cli/init.go (hunk 1 above)
Comment (sanderpick) [reply cmd/buck/cli/init.go:79]: We have those tests (added a month ago or so). It did compile, but `init` would throw a runtime error.

Comment (jsign) [reply cmd/buck/cli/init.go:79]: Ah, I read this diff wrong. Too early in the morning.

**Pull request merged**